pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
Also the centos slaves have pip 7.1, the first version that includes constraints.